packages/scene-composer/src/components/panels/SceneHierarchyPanel/components/SceneHierarchyTree/SceneHierarchyTreeItem.tsx

```diff
@@ -96,7 +115,12 @@ const SceneHierarchyTreeItem: FC<SceneHierarchyTreeItemProps> = ({
           {childNodes.map((node, index) => (
             <React.Fragment key={index}>
               {!isSearching && (
-                <SceneHierarchyTreeItem key={node.objectRef} enableDragAndDrop={enableDragAndDrop} {...node} />
+                <SceneHierarchyTreeItem
+                  key={node.objectRef}
+                  enableDragAndDrop={enableDragAndDrop}
+                  {...node}
+                  expanded={pathFromSelectedToRoot ? pathFromSelectedToRoot.indexOf(node.objectRef) > -1 : false}
```

Comment:
can be simplified to `pathFromSelectedToRoot.includes(node.objectRef)`

Reply:
I think the purpose is same just different approach. Also, the performance of both the approaches are identical. Is there any additional value other than coding style change?

Reply:
shorter code is more readable in general, especially for boolean

Reply:
Done